### PR TITLE
TS-399: add ES, IT and PT languages

### PR DESCRIPTION
**Pull request by tomas-maciulis**
Added languages and removed copyright templates from translation files

### CODE REVIEW

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+#: includes/class-wc-getkevin-settings.php:137
+msgid "Country select dropdown"
+msgstr "Selección de país en el menú desplegable"
+
+#: templates/getkevin-card.php:4
+msgid "Credit/Debit card"
+msgstr "Tarjeta de crédito/débito"
+
+#: includes/class-wc-getkevin-gateway.php:206
+msgid "Creditor account field must valid IBAN (Ex: LT01234567891011121314)."
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:74]: Our English translations was not even fixed before we moved to another languages, what a disappointment...

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+#: includes/class-wc-getkevin-settings.php:25
+msgid "Enable kevin."
+msgstr "Habilita a kevin."
+
+#: includes/class-wc-getkevin-settings.php:22
+msgid "Enable/Disable"
+msgstr "Activar/desactivar"
+
+#: includes/class-wc-getkevin-settings.php:65
+msgid "EndpointSecret"
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:96]: Would be good to add a space and name it `Endpoint Secret`

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+#: includes/class-wc-getkevin-settings.php:59
+msgid "Your Client Secret from kevin. dashboard."
+msgstr "Su secreto de cliente de kevin. salpicadero."
+
+#: includes/class-wc-getkevin-settings.php:68
+msgid "Your Endpoint Secret from kevin. dashboard."
+msgstr "Su secreto de punto final de kevin. tablero de instrumentos."
+
+#: includes/class-wc-getkevin-settings.php:86
+msgid "Your IBAN must be from the agreement with kevin.   "
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:353]: why do we have spaces at the end?

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+#: includes/class-wc-getkevin-settings.php:74
+msgid "Receiver name"
+msgstr "Nombre del receptor"
+
+#: includes/class-wc-getkevin-settings.php:118
+msgid "Redirect Preferred"
+msgstr "Redirección preferente"
+
+#: includes/class-wc-getkevin-settings.php:121
+msgid "Redirect user to make payments on bank side."
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:239]: on `a` bank side

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+msgstr "kevin. El estado del pago es %s"
+
+#: includes/class-wc-getkevin-gateway.php:565
+#: includes/class-wc-getkevin-gateway.php:566
+#, php-format
+msgid "kevin. Refund failed: %1$s"
+msgstr "kevin. Reembolso fallido: %1$s"
+
+#: includes/class-wc-getkevin-gateway.php:375
+msgid "kevin. Refund status changed to "
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:161]: is or has changed to

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+#, php-format
+msgid "kevin. Payment id is %s"
+msgstr "kevin. La identificación del pago es %s"
+
+#: includes/class-wc-getkevin-gateway.php:295
+msgid "kevin. Payment started."
+msgstr "kevin. El pago comenzó."
+
+#: includes/class-wc-getkevin-gateway.php:478
+msgid "kevin. Payment status changed to "
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:146]: is or has changed to

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 7]
```diff
@@ -0,0 +1,354 @@
...
+"kevin. es una empresa de infraestructuras de pago que ofrece servicios de "
+"iniciación de pagos en la EU y el EEE."
+
+#: includes/class-wc-getkevin-gateway.php:499
+#, php-format
+msgid "kevin. Payment id is %s"
+msgstr "kevin. La identificación del pago es %s"
+
+#: includes/class-wc-getkevin-gateway.php:295
+msgid "kevin. Payment started."
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:142]: has started

File: languages/woocommerce-gateway-getkevin-es_ES.po [hunk 8]
```diff
@@ -0,0 +1,354 @@
...
+
+#: includes/class-wc-getkevin-gateway.php:355
+#: includes/class-wc-getkevin-gateway.php:358
+msgid "kevin. Endpoint Secret is incorrect. Please check your kevin. settings."
+msgstr ""
+"kevin. El secreto del punto final es incorrecto. Por favor, compruebe la "
+"configuración de kevin."
+
+#: includes/class-wc-getkevin-gateway.php:216
+msgid "kevin. Exception: Bad client credentials"
```
Comment (saltanovas) [inline languages/woocommerce-gateway-getkevin-es_ES.po:125]: How this message is different from the `includes/class-wc-getkevin-gateway.php:99`? I do not really like the `kevin. Exception` beginning

File: includes/class-wc-getkevin-gateway.php [hunk 9]
```diff
@@ -582,7 +581,7 @@ public function process_refund($order_id, $amount = null, $reason = '')
     public function order_received_text($text, $order)
     {
         if ($order && $this->id === $order->get_payment_method()) {
-            $statusGroup = $_GET['statusGroup'] ?? null;
+            $statusGroup = isset($_GET['statusGroup']) ? $_GET['statusGroup'] : null;
```
Comment (saltanovas) [inline includes/class-wc-getkevin-gateway.php:584]: Could you tell me what's a default behaviour of the `$_GET` method when query parameter is not defined? Do we really need to convert the result to `null` if not defined?

**Review by saltanovas — CHANGES_REQUESTED** [review]

**Review by saltanovas — CHANGES_REQUESTED** [review]
Revert the [e203552](https://github.com/getkevin/kevin-woocommerce/pull/17/commits/e203552f85e0b1d9624c579fe50b7d45283446e4) commit and we will wait for missing base English translations to the ES, IT and PT languages :)

**Review by saltanovas — CHANGES_REQUESTED** [review]
Language keys was changed but not applied to the code? The complexity of the pull request has grown a lot... :(

**Review by KubaJM — APPROVED** [review]

**Review by saltanovas — APPROVED** [review]

**Pull request merged**